viewbuilder-android/build.gradle

```diff
@@ -15,6 +15,10 @@
  */
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
+apply plugin: 'com.vanniktech.maven.publish'
+
+sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
Out of scope, but should we be bumping this to 1_8?

Reply:
I guess we have no real reason to do so.

Reply:
Yea, probably.